ui/app/templates/workspace/projects/index.hbs

```diff
@@ -1,54 +1,78 @@
-{{page-title "Projects"}}
+{{page-title 'Projects'}}
 
 {{#if this.cli}}
-<div class="flash flash--success">
-  <div class="flash-header">
-    <FlightIcon @name="check-circle-fill" class="icon" />
-    <p>Authenticated successfully via the Waypoint CLI</p>
+  <div class='flash flash--success'>
+    <div class='flash-header'>
+      <FlightIcon @name='check-circle-fill' class='icon' />
+      <p>Authenticated successfully via the Waypoint CLI</p>
+    </div>
   </div>
-</div>
 {{/if}}
 
-<PageHeader @className="projects-header">
-  <div class="title">
+<PageHeader @className='projects-header'>
+  <div class='title'>
     <h1>Projects</h1>
     <small>
       {{#if @model.length}}
-      {{pluralize @model.length "project"}}
+        {{pluralize @model.length 'project'}}
       {{else}}
-      No projects created yet
+        No projects created yet
       {{/if}}
     </small>
   </div>
-  <Pds::CtaLink @route="workspace.projects.create" @variant="secondary "class="pds--iconStart">
-    <FlightIcon @name="plus" class="pds-button__iconStart"/>
-    &nbsp;{{t "form.project_create.title"}}
+  <Pds::CtaLink @route='workspace.projects.create' @variant='secondary ' class='pds--iconStart'>
+    <FlightIcon @name='plus' class='pds-button__iconStart' />
+    &nbsp;{{t 'form.project_create.title'}}
   </Pds::CtaLink>
 </PageHeader>
 
+<Hds::Alert class='deprecation-banner' @type='inline' @color='neutral' @icon='info' as |A|>
+  <A.Title>
+    {{t 'page.project.deprecation-banner.title'}}
+  </A.Title>
+  <A.Description>
+    <div class='deprecation-description'>
+      {{t 'page.project.deprecation-banner.description'}}
+      <div>
+        <a
+          target='_blank'
+          rel='noopener noreferrer'
+          href='https://www.waypointproject.io'
+          class='deprecation-external-link'
+        >
```

Comment:
Could we use [`<Hds::Link::Standalone>`](https://design-system-components-hashicorp.vercel.app/components/link/standalone) for this?

Reply:
I didn't know about that component, that's definitely preferable. Also going to do it on HCP so you don't have to comment there as well 🙂